tests/accessor/accessor_exceptions.cpp

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for accessor exceptions.
+//
+// This test provides verifications for generic accessor, host_accessor and
+// local_accessor.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "../common/disabled_for_test_case.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+
+#include "accessor_exceptions.hpp"
+
+using namespace accessor_tests_common;
+#endif
+
+namespace accessor_exceptions_test {
+using namespace sycl_cts;
+
+DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
+("Generic sycl::accessor constructor exceptions test.", "[accessor]")({
+  typedef std::integral_constant<accessor_type, accessor_type::generic_accessor>
+      generic_accessor;
+#ifndef SYCL_CTS_ENABLE_FULL_CONFORMANCE
+  const auto types = get_lightweight_type_pack();
+#else
+  const auto types = get_full_conformance_type_pack();
+#endif  // SYCL_CTS_ENABLE_FULL_CONFORMANCE
+  for_all_types_vectors_marray<run_tests_with_types, generic_accessor>(types);
```

Comment:
Since we checking accessor exceptions for different types, maybe tests for fp64 and fp16 also could be added?

Reply:
I've added verifications for fp64 and fp16 types in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/329/commits/80942e5c1cabb71e4559f242ad2c8dd48cb557a2